include/emitc/emitc_core_ops.h

```diff
@@ -314,6 +325,78 @@ inline Dest reshape(Src x) {
   return z;
 }
 
+// SliceOp
+// Overload for 1d case.
+template <typename Dest, typename Src, IsTensorOfDim<1, Src> = true>
+Dest slice(Src x, Tensor<int64_t, 1> start_indices,
+           Tensor<int64_t, 1> limit_indices, Tensor<int64_t, 1> strides) {
+  Dest z;
+
+  size_t index = 0;
+  for (int64_t i = start_indices[0]; i < limit_indices[0]; i += strides[0]) {
```

Comment:
Hm, to you really want a mishmash of `size_t` and `int64_t`?

Reply:
What do you mean? All variables are `int64_t`

Reply:
`index` is a `size_t`, so depending on the platform probably a unsigned 64-bit integer. That's fine for me, but I am not sure if is useful that the variable `i` is signed.

Reply:
Variable `i` is signed, because the `{start,limit}_indices` and `strides` are signed. And these are signed, because MHLO specifies the slice op like that, where the slice implementation is taken from.

Reply:
Even worse, `size_t` can be 32-bit. Let's change this to `unit64_t` :)

Reply:
We decided internally that we keep it as `size_t` for the moment, since the Tensor class' bracket operator uses `size_t` as well. A general overflow detection technique might be needed in general in the future.